scionlab/scion_config.py

```diff
@@ -57,21 +57,15 @@
     SCIOND_API_SOCKDIR,
 )
 
-SERVICE_TYPES_CONTROL_PLANE = [Service.BS, Service.CS, Service.PS]
-SERVICE_TYPES_SERVER_APPS = [Service.BW, Service.PP]
-
 TYPE_BR = 'BR'
 TYPE_SD = 'SD'
```

Comment:
`TYPE_SD` can be removed now (and with my suggestion below, also `TYPE_BR`).

Reply:
I don't think so, it seems these values are still used in the `_write_supervisord_files()` function

Reply:
Ugh, ok sorry.